service_container/tags.rst

```diff
@@ -405,3 +405,92 @@ The double loop may be confusing. This is because a service can have more
 than one tag. You tag a service twice or more with the ``app.mail_transport``
 tag. The second foreach loop iterates over the ``app.mail_transport``
 tags set for the current service and gives you the attributes.
+
+Reference tagged services
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case your tag doesn't require any further additional attributes writing compiler 
+passes per tag might become tedious. A way to overcome this is is to make your compiler 
+pass more generic. The downside of this approach is you have to write and maintain
+additional code, considering you want to reuse it over multiple projects.
+
+ThereBecause this task is so generic and common to do, Symfony provides a way to achieve this 
+directly in your service container confguration. This enables to inject services tagged 
+with e.g. `app.handler` into another service that collects all handlers.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        services:
+            App\Handler\One:
+                tags: [app.handler]
+
+            App\Handler\Two:
+                tags: [app.handler]
+
+            App\HandlerCollection:
+                arguments: [!tagged app.handler]
```

Comment:
Let's put a comment here describing that this is passing all services tagged with `app.handler` as an array argument to this class. (Same for the other formats)